simple-csweb.user.css

```
@@ -8,6 +8,100 @@
 @var checkbox show-right-sidebar "Show Right Sidebar" 0
 ==/UserStyle== */
 
+@-moz-document regexp("https://csweb.u-aizu.ac.jp/campusweb/campusportal.do(\?locale.*|)") {
```

```suggestion
@-moz-document regexp("https://csweb.u-aizu.ac.jp/campusweb/campusportal.do(?!.*page=).*") {
```
厳密に `page=` の指定がないときのみスタイルを適用します